### PR TITLE
Updates to bin/update_readme script to fit current Symfony version

### PR DESCRIPTION
**Pull request by Jampire**
Hi. I updated bin/update_readme to fit current Symfony versions and re-generated README.md file applying these changes. Also I included PR #226 as there is no sense to update README.md file only.

### CODE REVIEW

File: bin/update_readme [hunk 1]
```diff
@@ -38,7 +38,7 @@ $sectionTemplate = <<<EOF
         %PROVIDER_NAME%:
             # must be "%PROVIDER_NAME%" - it activates that type!
             type: %PROVIDER_NAME%
-            # add and configure client_id and client_secret in parameters.yml
+            # add and configure client_id and client_secret in parameters.yml or .env* files
```
Comment (weaverryan) [inline bin/update_readme]: Let's go full "new Symfony" with this :) 

```suggestion
            # add and set these environment variables in your .env files
```

**Comment by weaverryan** [comment]
Great catch and fix for these outdated docs! Awesome! I tweaked a bit further in sha: 1483e391ef71ca5558fb1ddf9e2fec97dd248474 to only show the "new" way of doing things.

Thanks!

File: bin/update_readme (hunk 1 above)
Comment (Jampire) [reply bin/update_readme]: Hi @weaverryan , I thought about this initially. But I noticed that `oauth2-client-bundle` supports Symfony v3.4. And this version still uses parameters.yml.

Comment (weaverryan) [reply bin/update_readme]: It does - but you're own your own as far as documentation goes :). It's impossible to write good documentation for multiple versions at once. For example, just mentioning parameters.yml here isn't enough - 3.4 doesn't use the `env()` syntax at all! Better to be clear and on the latest version

**Pull request merged**